x-pack/plugins/apm/public/context/apm_service/apm_service_context.tsx

```diff
@@ -63,6 +67,16 @@ export function ApmServiceContextProvider({
     agentName,
   });
 
+  // Replace transactionType in the URL in case it is not one of the types returned by the API
+  useMemo(() => {
```

Comment:
Shall we use `useEffect` here?

Reply:
Removed it completely here: https://github.com/elastic/kibana/pull/136569/commits/71613a072c0269b6991cf0b80815eb447d636752